outlines/structure.md

```diff
@@ -0,0 +1,61 @@
+# Application structure and code style
+
+1. What's different between Meteor and other kinds of apps?
+    1. All JavaScript, enables code sharing
+    2. Different envrironment: Client vs. server vs. both
+    3. Module system is new, but we recommend its use (answer to why `imports/`?)
+2. General JavaScript structure
+    1. All code in `imports/` apart from initializers in `main/`
+    2. Group code into data - `api/` and rendering - `client` (?)
+    1. Directory structure around features, not client/server
+    2. LESS/SCSS files are in the same directory as components
+    3. One file per unit - template, method, collection, etc
```

Comment:
We should mention tests as well, to emphasize that 1. you should have tests and 2. you should put them next to the thing they test